tests/drivers/clock_control/nrf_clock_calibration/src/test_nrf_clock_calibration.c

```diff
@@ -63,6 +63,10 @@ static void test_calibration(uint32_t exp_cal, uint32_t exp_skip,
 	int cal_cnt;
 	int skip_cnt;
 
+	const struct device *const clk_dev = DEVICE_DT_GET_ONE(nordic_nrf_clock);
+
+	turn_on_clock(clk_dev, CLOCK_CONTROL_NRF_SUBSYS_HF);
```

Comment:
enabling and disabling HF clock is part of LF clock calibration. IMO it would be better to prolong CALIBRATION_PROCESS_TIME_MS for SoC with XOTUNE (nRF54L series?).

Reply:
however time of XOTUNE event occurrence may vary. As it is test of LF calibration (not HF startup) I would prefer to keep it as it is